modules/gdscript/gdscript_compiler.cpp

```diff
@@ -643,7 +645,7 @@ GDScriptCodeGenerator::Address GDScriptCompiler::_parse_expression(CodeGen &code
 							self.mode = GDScriptCodeGenerator::Address::SELF;
 							MethodBind *method = ClassDB::get_method(codegen.script->native->get_name(), call->function_name);
 
-							if (_can_use_validate_call(method, arguments)) {
+							if (method && _can_use_validate_call(method, arguments)) {
```

Comment:
I'm not sure about this, it still passes `method` to `gen->write_call_method_bind(result, self, method, arguments);`, what does it do with it? What happens with it down the line?

Reply:
I have tested it before when `_can_use_validate_call` returned false and it was safe to call `gen->write_call_method_bind(result, self, method, arguments);` with nullptr as method.

Reply:
I'd say the real issue is that `has_method` returns true but `get_method` doesn't return a valid pointer, so something here is wrong elsewhere unless some of the checking is done incorrectly, I think the `method == nullptr` is just a symptom of something else being wrong here

Reply:
> I have tested it before when `_can_use_validate_call` returned false and it was safe to call `gen->write_call_method_bind(result, self, method, arguments);` with `nullptr` as method.

That probably just means the method is never actually called, or attempted, the code in the VM should segfault or behave poorly as there's no check for the nullptr AFAIK

Reply:
The issue only happens when importing gdscript resources for the first time, maybe to check for errors to throw them after loading the project, in my custom build all the missing classes exists, only was using `Image.create` instead of `Image.create_empty` and even changing it didn't prevent the crash. I can't find why this happened for nearly 8 scripts in the project. Also running the game or opening those scripts after the first crash without deleting the `.godot` folder causes another crash.

Reply:
I'm just wondering why importing the script in 4.2 first doesn't cause crash after opening the project again with 4.x, maybe it's related to how the editor is caching the scripts.